fixtures/odd_stub/gen.py

```diff
@@ -4,21 +4,21 @@
 import os
 import sys
 
-random = Random(275)
-directory = sys.argv[1]
+if len(sys.argv) == 1:
+    print("01 repeat=10")
```

Comment:
One of the main pros of the new generator format is the ability to have named inputs. The fixtures also serve as examples, so we should encourage good practice and name our inputs. How about `random` of `random-uniform`?